cmake/cpu_extension.cmake

```diff
@@ -82,15 +85,40 @@ else()
     message(FATAL_ERROR "vLLM CPU backend requires AVX512 or AVX2 or Power9+ ISA support.")
 endif()
 
+#
+# Build oneDNN for W8A8 GEMM kernels (only for x86-AVX512 platforms)
```

Comment:
Yes, this dnnl is only used for the W8A8 GEMM, and related functions (quant, dequant, ...) are implemented with avx512 only for now. So it will not be built when avx512 is disabled.